extensions/canvasMedical/v1/validation/task.zod.ts

```diff
@@ -42,7 +42,7 @@ export const taskSchema = z.object({
   restriction: z
     .object({
       period: z.object({
-        end: z.string().datetime(),
+        end: z.string(),
```

Comment:
If it's really a date I don't think it should be removed.  If that's zod problem with date parsing, check `DateTime` and `DateOnly` validators from `extensions-core for dates instead, they use `date-fns` library for validation and should be more accurate in identifying different formats.

Reply:
DateTimeSchema :O